src/System.Management.Automation/engine/PSConfiguration.cs

```diff
@@ -133,12 +174,10 @@ internal void SetExecutionPolicy(ConfigScope scope, string shellId, string execu
                 // CreateDirectory will succeed if the directory already exists
                 // so there is no reason to check Directory.Exists().
                 Directory.CreateDirectory(appDataConfigDirectory);
```

Comment:
If we decided to get rid of `appDataConfigDirectory`, then it needs to be changed to `Path.GetDirectoryName(appDataConfigFile)` here.

Reply:
Noted.